iroh/src/client.rs

```diff
@@ -1,18 +1,24 @@
 //! Client to an Iroh node.
 
 use futures_lite::{Stream, StreamExt};
-use quic_rpc::{RpcClient, ServiceConnection};
+use quic_rpc::RpcClient;
 use ref_cast::RefCast;
 
 #[doc(inline)]
 pub use crate::rpc_protocol::RpcService;
 
-mod mem;
 mod quic;
 
-pub use self::mem::{Doc as MemDoc, Iroh as MemIroh, RpcClient as MemRpcClient};
+// #[deprecated]
+// pub use self::docs::Doc as MemDoc;
+// #[deprecated]
+// pub use self::Iroh as MemIroh;
+// #[deprecated]
+// pub use self::docs::Doc as QuicDoc;
+// #[deprecated]
+// pub use self::Iroh as QuicIroh;
```

Comment:
did you mean to remove these or leave them in?

Reply:
I wanted to leave them in. I just commented them out to remove all useages because rustc did not scream at me due to the deprecation warnings. I think they don't work for reexports.